Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -439,38 +440,41 @@ - (NSString *)createNewLogFileWithError:(NSError *__autoreleasing  _Nullable *)e
 
     NSString *fileName = [self newLogFileName];
     NSString *logsDirectory = [self logsDirectory];
-    NSData *fileHeader = [self logFileHeaderData];
-    if (fileHeader == nil) {
-        fileHeader = [NSData new];
-    }
+    NSData *fileHeader = [self logFileHeaderData] ?: [NSData new];
 
+    NSString *baseName = nil;
```

Comment:
These (`baseName` and `extension`) can be declared inline below.

Reply:
(Inside the `if (attempt > 1)`)

Reply:
No because I want to lazily cache the result of `[fileName stringByDeletingPathExtension]` and `[fileName pathExtension]`.

Reply:
Good point.